src/blocks/notify.rs

```diff
@@ -16,6 +16,10 @@
 //! `icon`      | Icon based on notification's state         | Icon   | -
 //! `paused`    | Present only if notifications are disabled | Flag   | -
 //!
+//! Action   | Default button
+//! ---------|---------------
+//! `toggle` | Left
```

Comment:
I'd suggest {,toggle_}pause, {,toggle_}dnd, or {,toggle_}do_not_disturb since we may want to add other types of (togglable?) actions.